File: app/src/main/java/ua/meugen/android/pdfcomposer/model/PdfExporter.java
package ua.meugen.android.pdfcomposer.model;

import java.util.List;

import ua.meugen.android.pdfcomposer.model.data.PageContent;

public interface PdfExporter {

    void exportPages(List<PageContent> pages);

    void onPdfExportSuccess();

    void onPdfExportError(Throwable th);
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/model/PdfExporter.kt
package ua.meugen.android.pdfcomposer.model

import ua.meugen.android.pdfcomposer.model.data.PageContent

interface PdfExporter {

    fun exportPages(pages: List<PageContent>)

    fun onPdfExportSuccess()

    fun onPdfExportError(th: Throwable)
}
